index.rst

```diff
@@ -36,70 +44,80 @@
 
    Feel free to delete this instructional comment.
 
-:tocdepth: 1
 
-.. Please do not modify tocdepth; will be fixed when a new Sphinx theme is shipped.
 
-.. sectnum::
 
 
 
 Introduction
 ============
 
-Adding a filter or disperser to LATISS is a multi-step procedure that spans the DM and T&S subsystems.
+Changing a filter or disperser in LATISS requires a few steps to make sure the instrument is properly configured and check data ingestion. 
+Adding new, previously uninstalled a filter or disperser to LATISS is a multi-step procedure that spans the DM and T&S subsystems.
```

Comment:
Suggested rephrase:

> When adding new, previously uninstalled filter or disperser requires additional steps that spans both DM and T&S subsystems.
> Some of these steps may take some time to be ready to be rolled out to the summit and should be done well in advance to prevent delaying operations.